src/Components/Support/Cron.php

```diff
@@ -13,12 +13,20 @@ class Cron extends BladeComponent
     /** @var string */
     public $schedule;
 
+    /** @var string */
+    public $locale = 'en';
+
+    /** @var bool */
+    public $use24hour = false;
+
     /** @var bool */
     public $human = false;
 
-    public function __construct(string $schedule, bool $human = false)
+    public function __construct(string $schedule, string $locale = 'en', bool $use24hour = false, bool $human = false)
```

Comment:
I haven't had time yet to thoroughly review this but you'll have to place the new arguments here at the end of the constructor call. Otherwise it's a breaking change for anyone manually instantiating this object.

Reply:
Good point, that slipped my mind.

Fixed, thanks!